deploy/deploy.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-oc apply -f https://raw.githubusercontent.com/openshift/sandboxed-containers-operator/master/deploy/deploy.yaml
+oc apply -f https://raw.githubusercontent.com/openshift/sandboxed-containers-operator/release-4.8/deploy/deploy.sh
```

Comment:
This part, with explicit `4.8`, is a bit inconsistent with `latest` above.

Reply:
agreed

Reply:
thinking about it. the release-4.8 is just the branch name. I think that's okay, the yaml file on that branch will always use 4.8-latest. But I introduced a bug here changing deploy.yaml to deploy.sh, which is nonsense :-)

Reply:
fixed now, thank you @c3d!